imports/plugins/core/graphql/server/no-meteor/util/applyPaginationToMongoCursor.js

```diff
@@ -26,6 +26,9 @@ export default async function applyPaginationToMongoCursor(cursor, { first, last
   const limit = first || last || DEFAULT_LIMIT;
 
   let skip = 0;
+
+  if (offset) skip = offset;
```

Comment:
I'd prefer not to mix the two methods in this one file. I think there should be two options:
- If `offset` param is present, pass that and an optional `limit` param to a different version of this `applyPaginationToMongoCursor` function (maybe `applyOffsetPaginationToMongoCursor`). This function would be much simpler, needing far less logic to determine if there's a previous or next page.
- Otherwise if `offset` param is NOT present, fall back to using this function without any changes as before.
- If `offset` or `limit` are present AND one of the existing four paging params are also present, throw an error. They can't mix-n-match.

Reply:
@aldeed this is just adhering to the examples in the GraphQL pagination documentation: https://graphql.org/learn/pagination/

It would seem to complicate things to have pagination handled in multiple methods since these arguments basically enable the client to determine their pagination implementation. I can certainly separate the pagination methods into their opinionated methods, but worry that complicates the logic. Let me know if this is a hard request for this PR.